public/app/plugins/datasource/loki/querybuilder/operations.ts

```diff
@@ -238,6 +238,28 @@ export function getOperationDefinitions(): QueryBuilderOperationDef[] {
       addOperationHandler: addLokiOperation,
       explainHandler: (op) => `Return log lines that contain string \`${op.params[0]}\`.`,
     },
+    {
+      id: LokiOperationId.LineContainsInsensitive,
```

Comment:
I'd update the order of these operations. As you can see bellow, they are usually grouped (`contains` and then `not contains` under each other). But in this case, they are not. Also, this is probably just personal preference/nit pick, but I'd go with `LineContainsCaseInsensitive` operation name. 

<img width="352" alt="image" src="https://user-images.githubusercontent.com/30407135/200323429-01646b79-31a5-4c79-99e9-7fd5ec8f9668.png">